lighthouse-core/test/fixtures/budgets.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
we'll probably end up with multiple budget fixtures, so maybe `test/fixtures/budgets/simple-budget.json` (or a better name than that for the file :)